sections/discussion/discussion-chapter.tex

```diff
@@ -80,16 +80,16 @@ \subsubsection{Challenges with utilizing context}
 This is especially prominent in the \textit{CAMF-CI} approach as the amount of model parameters scales with the amount of items in the dataset.
 
 \subsubsection{Did context and side information matter?}
-As reported in \autoref{subsec:resultsofexperiment}, \textit{IS-UserBased} showed the highest recall score for the LDOS-CoMoDa and MovieLens datasets, which are the ones with contextual information.
-However, at the same time it had the second worst precision, which is most likely due to the precision/recall trade-off discussed in \autoref{subsub:precrectradeoff}.
-Due to the very low precision score, it ends up with the third worst F1 score of the 11 methods, only surpassing LightGCN and KGAT.
-To do a more complete comparison, the method was also run using side-information for item splitting for the Yahoo dataset, in which case it once again had the best recall score, and the third lowest precision.
-In this case, it outperformed NGCF and KGAT for the precision score.
-However, using side-information instead of context it ends up outperforming both NGCF, KGAT, LightGCN and Random in F1 scores.\\
-Looking at the other context-based methods, \textit{CAMF-C} and \textit{CAMF-CI}, they end up with respectively a second and forth place in the F1 metric on the LDOS-CoMoDa dataset, only being outperformed by SVD and SVD++.\\
-For MovieLens, \textit{CAMF-C} gets a first place on F1, and \textit{CAMF-CI} on a fifth place, being outperformed by \textit{SVD, SVD++} and \textit{DeepWalk}.\\
-Looking at the results in general, the \textit{IS-Userbased} method does well on recall, and \textit{CAMF-C(I)} does well on F1 scores, which bodes well for using context and side-information.\\
-The \textit{MAP@10} results are a slightly more inconclusive on the other hand, we see that the \textit{IS-UserBased} method achieves the best results on the Yahoo dataset, while the \textit{CAMF-C(I)} methods perform very poorly on the same dataset.
+As reported in \autoref{subsec:resultsofexperiment}, \textit{IS-UserBased} showed the highest Recall@10 score for the LDOS-CoMoDa and MovieLens datasets, which are the ones with contextual information.
```

Comment:
hvis vi konsekvent skriver textit til metode navne skulle vi så overveje at gøre det for datasæt navne?